### PR TITLE
fix(ci): bump markdownlint

### PR DESCRIPTION
**Pull request by lukashroch**
Update markdownlint:
- fixes dead links
- re-lint *.MD files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,38 +3,60 @@
 [![Build Status](https://github.com/RobinBuschmann/sequelize-typescript/workflows/Node.js%20CI/badge.svg)](https://github.com/RobinBuschmann/sequelize-typescript/actions?query=workflow%3A%22Node.js+CI%22)
```
Comment (WikiRik) [inline README.md:3]:
```suggestion
[![Build Status](https://github.com/sequelize/sequelize-typescript/workflows/Node.js%20CI/badge.svg)](https://github.com/sequelize/sequelize-typescript/actions?query=workflow%3A%22Node.js+CI%22)
```

File: README.md [hunk 2]
```diff
@@ -3,38 +3,60 @@
...
 [![codecov](https://codecov.io/gh/RobinBuschmann/sequelize-typescript/branch/master/graph/badge.svg)](https://codecov.io/gh/RobinBuschmann/sequelize-typescript)
```
Comment (WikiRik) [inline README.md:4]:
```suggestion
[![codecov](https://codecov.io/gh/sequelize/sequelize-typescript/branch/master/graph/badge.svg)](https://codecov.io/gh/sequelize/sequelize-typescript)
```

File: README.md [hunk 3]
```diff
@@ -250,11 +272,11 @@ Get/set accessors do work as well
 class Person extends Model {
```
Comment (WikiRik) [inline README.md:272]: Can't put a comment here, but also update the link in line 247 (@Unique decorator)

**Review by WikiRik — COMMENTED** [review]
Forgot these three

File: README.md (hunk 3 above)
Comment (lukashroch) [reply README.md:249]: Thanks fixed!

File: README.md (hunk 2 above)
Comment (lukashroch) [reply README.md]: Thanks fixed!

File: README.md (hunk 1 above)
Comment (lukashroch) [reply README.md]: Thanks fixed!

**Review by WikiRik — APPROVED** [review]

**Pull request merged**